### DISCUSSION TITLE
Copilot gone even if linked to university email

### DISCUSSION THREAD

**Discussion by Tebarem** [community/community#40783] [Copilot Conversations]
### Select Topic Area

Question

### Body

Hey guys I made a ticket about this yesterday and when I checked just now the ticket is gone. I was buying the yearly subscription of copilot but later on realized I haven't been charged. I later check and see that it says University students get copilot for free. Later on after about 3-5 months of using copilot yesterday it was gone and don't know why.

**Comment by davecheney** [comment]
@e1pupper copilot is available to students (or faculty) with a current student (or faculty) coupon active on their account. I've checked your account and I cannot see a student coupon active on your account.

You can apply for a student coupon here, https://education.github.com/pack, once it is active on your account, if you sign up for copilot it will show the option for free access.

**Reply by davecheney** [reply]
@e1pupper further, it appears the last time you used this account to access copilot, before today, was back in June. Is it possible you were using a different account up until recently?

**Reply by Tebarem** [reply]
no I was using copilot on vscode and last time I was using it was around last 2-3 weeks

> @e1pupper further, it appears the last time you used this account to access copilot, before today, was back in June. Is it possible you were using a different account up until recently?

**Reply by Tebarem** [reply]
> @e1pupper copilot is available to students (or faculty) with a current student (or faculty) coupon active on their account. I've checked your account and I cannot see a student coupon active on your account.
> 
> You can apply for a student coupon here, https://education.github.com/pack, once it is active on your account, if you sign up for copilot it will show the option for free access.

I have signed up for this and it is why my university email is linked in the first place. I will try and do it again and update you. Thanks for the response

**Reply by Tebarem** [reply]
now that i think about it. It might be cause my student id has expired I might check that out
